src/qt/clientmodel.cpp

```diff
@@ -235,7 +246,7 @@ static void BannedListChanged(ClientModel *clientmodel)
     assert(invoked);
 }
 
-static void BlockTipChanged(ClientModel *clientmodel, bool initialSync, int height, int64_t blockTime, double verificationProgress, bool fHeader)
+static void BlockTipChanged(ClientModel *clientmodel, bool initialSync, const uint256 blockHash, int height, int64_t blockTime, double verificationProgress, bool fHeader)
```

Comment:
In commit "Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals." (4948d4ee50d9960b23f43e14938845b0bb21772b)

Maybe use reference type `const uint256&` to be a little more efficient and avoid a copy here

Reply:
done